wdl/pipelines/TechAgnostic/Visualization/SvQCPlots.wdl

```diff
@@ -0,0 +1,451 @@
+version 1.0
+
+import "../../../structs/Structs.wdl"
+
+workflow PlotSVQCMetrics{
+
+    meta{
+        description: "This workflow generates an SV stats summary for the provided callers and their generated VCFs. It then concatenates and plots the SV stats."
+    }
+    parameter_meta{
+        gcs_vcf_dir: "GCS path to directory containing VCFs. Files should be named <sample_name>.<caller>.vcf.gz"
+        samples: "List of sample names"
+        coverage_metrics: "List of coverage metrics for each sample"
+        callers: "List of SV callers"
+        reference_name: "Reference genome name"
+    }
+
+    input{
+        String gcs_vcf_dir
+        Array[String] samples
+        Array[Float] coverage_metrics
+        Array[String] callers
+        String reference_name = "GRCh38"
+    }
+
+    scatter(caller in callers){
+        scatter(sample in samples){
+            call bcfQuerySV{
+                input:
+                    sample_name = sample,
+                    input_vcf = gcs_vcf_dir + "/" + sample + "." + caller + ".vcf.gz",
+                    caller = caller,
+            }
+        }
+    }
+    Array[File] all_SV_stats = flatten(bcfQuerySV.all_SV_stat_out)
+
+    call concatSVstats{
+        input:
+            all_stats = all_SV_stats,
+            callers = callers,
+    }
+
+    call compileSVstats{
+        input:
+            samples = samples,
+            all_stats = all_SV_stats,
+            callers = callers,
+    }
+    call addCoverageToSVstats{
+        input:
+            coverage_stats = coverage_metrics,
+            samples = samples,
+            allStatsBySample = compileSVstats.allStatsBySample,
+            callers = callers,
+    }
+
+    call plotSVQCMetrics{
+        input:
+            all_stats_with_cov = addCoverageToSVstats.all_stats_with_cov,
+            all_stats_by_type = concatSVstats.all_stats_by_type,
+            callers = callers,
+            reference_name = reference_name,
+    }
+
```

Comment:
consider adding tasks that copy the outputs to a designated location. 
This way outputs won't be lost when the `submission` folder is cleared.

Reply:
done by SHuang branch merger